README.md

```diff
@@ -65,7 +65,7 @@ oneMath is part of the [UXL Foundation](http://www.uxlfoundation.org).
             <td align="center">AMD GPU</td>
         </tr>
         <tr>
-            <td align="center"><a href="https://github.com/codeplaysoftware/portBLAS"> portBLAS </a></td>
+            <td align="center"><a href="https://github.com/codeplaysoftware/portBLAS"> generic SYCL BLAS </a></td>
```

Comment:
There many links that still mention and bring to portBLAS codebase. I will update them as soon as the migration is completed.

Reply:
Updated in [2abd2c7](https://github.com/uxlfoundation/oneMath/pull/618/commits/2abd2c7eb9ba7a90e0f93c8166c3df8fe5652ec4)